work_items/schema_directory.md

```diff
@@ -0,0 +1,30 @@
+# Schema Directory
+
+The DIF Schema Directory will act as a source of curated references to aid 
+developers in finding schemas that address their needs. When seeking to add 
+to the directory, the group will first seek out existing schemas from external 
+sources that cover the motivating need for the addition. If no schema exists 
+externally that addresses the need, the group may elect to generate and publish 
+new schemas and publish them within the directory project/repo.
+
+## Work Item Owners
+- Daniel Buchner (@github, email@example.com, or both)
+- Martin Riedel (@rado0x54)
```

Comment:
Would @wyc / spruce be interested to join?

Reply:
and @OR13

Reply:
@csuwildcat Ideally we find another owner.

Reply:
2 NON-CHAIR codeowners should be more explicit as a req :D

Reply:
welcome aboard, @JackSophtron !